bool get_gate
---
string entrance
string exit